fetch.bs

```diff
@@ -5225,13 +5330,17 @@ these steps:
   <a>cross-origin resource policy check</a> with <var>request</var>'s <a for=request>origin</a>,
   <var>request</var>'s <a for=request>client</a>, <var>request</var>'s
   <a for=request>destination</a>, and <var>internalResponse</var> returns <b>blocked</b>, then
-  return a <a>network error</a>.
+  return the result of running <a>HTTPS upgrade fallback</a> given <var>request</var>
+  and a <a>network error</a>.
 
   <p class=note>The <a>cross-origin resource policy check</a> runs for responses coming from the
   network and responses coming from the service worker. This is different from the
   <a>CORS check</a>, as <var>request</var>'s <a for=request>client</a> and the service worker can
   have different embedder policies.
 
+ <li>Set <var>response</var> and <var>internalResponse</var> to the result of running
```

Comment:
This is a bit confusing. It might be better to externalize the condition from the fallback algorithm or rename it to clarify that it doesn't fallback unconditionally.

Reply:
Done